### ISSUE TITLE
Can't register custom fields to form builder

### ISSUE THREAD

**Issue by corneliusio** [verbb/formie#114]
**Description**
I'm wagering this is likely either something missing from the docs or me missing it in the docs (I've read through https://verbb.io/craft-plugins/formie/docs/developers/custom-field)

But, I put together a small Stripe integration to meet a client need that required a couple custom fields. I was able to get it working but feel like I had to use a pretty hacky way to get the custom fields to be availble to the edit form screen.

Added the custom fields as outlined in the documentation:
```php
Event::on(
    Fields::class,
    Fields::EVENT_REGISTER_FIELDS,
    function(RegisterFieldsEvent $event) {
        $event->fields[] = StripeElement::class;
        $event->fields[] = StripeProducts::class;
    }
);
```
These field did seem to initialize after doing this, however they did not show up on the edit form screen. I ended up having to add the following code to use to them:
```php
if (Craft::$app->request->getIsCpRequest()) {
    Event::on(
        View::class,
        View::EVENT_BEFORE_RENDER_TEMPLATE,
        function (TemplateEvent $event) {
            if ($event->template === 'formie/forms/_edit') {
                $event->variables['fields'][2]['fields'][] = Formie::$plugin
                    ->getFields()
                    ->getRegisteredField(StripeElement::class)
                    ->getBaseFieldConfig();
                $event->variables['fields'][2]['fields'][] = Formie::$plugin
                    ->getFields()
                    ->getRegisteredField(StripeProducts::class)
                    ->getBaseFieldConfig();
            }
        }
    );
}
```

Am I just completely missing the right way to do this in the docs? Or is there another step that isn't documented?

**Additional info**
- Plugin version: 1.2.14
- Craft version: 3.5.11.1
- Multi-site: No

**Comment by engram-design** [comment]
When you say they didn't show up - as in they didn't appear in the right-hand sidebar for you to add to your form?

**Comment by corneliusio** [comment]
That's right. The custom fields do initialize, however they don't seem to get registered to the sidebar.

**Comment by engram-design** [comment]
Should be fixed in [1.2.15](https://github.com/verbb/formie/releases/tag/1.2.15).

**Comment by corneliusio** [comment]
Working exactly as expected now, thanks a ton!
